cmd/observe/flows.go

```diff
@@ -200,17 +200,79 @@ func newFlowsCmdWithFilter(vp *viper.Viper, ofilter *flowFilter) *cobra.Command
 
   Then,
 
-    cat flows.json | hubble observe
+    cat flows.json | hubble observe --input-file -
 
   Note that the observe command ignores --follow, --last, and server flags when it
   reads flows from stdin. The observe command processes and output flows in the same
-  order they are read from stdin without sorting them by timestamp.`,
+  order they are read from stdin without sorting them by timestamp.
+
+* Filtering flows
+
+  Observe provide a long list of filter options. These option let us, for example,
```

Comment:
Fix grammar, prefer `you` over `we`.

```suggestion
  Observe provides a long list of filter options. These options let you, for example,
```